datacenter/ucp/2.2/guides/admin/install/system-requirements.md

```diff
@@ -9,23 +9,34 @@ Before installing, be sure your infrastructure has these requirements.
 
 ## Hardware and software requirements
 
-You can install UCP on-premises or on a cloud provider. To install UCP,
-all nodes must have:
+You can install UCP on-premises or on a cloud provider. Common requirements:
 
 * [Docker Enterprise Edition](/engine/installation/index.md) version 17.06 or higher
 * Linux kernel version 3.10 or higher
+* A static IP address
+
+### Minimum requirements
+
 * 8GB of RAM for manager nodes or nodes running DTR
 * 4GB of RAM for worker nodes
 * 3GB of free disk space
-* A static IP address
+
+### Recommended production requirements
+
+ * 16GB of RAM for manager nodes or nodes running DTR
+ * 4 vCPUs for manager nodes or nodes running DTR
+ * 25-100GB of free disk space
+ 
+Note that Windows container images are typically larger than Linux ones and for that reason, you should consider provisioning more local storage for Windows nodes and for DTR setups that will store Windows container images.
 
 Also, make sure the nodes are running one of these operating systems:
 
-* A maintained version of CentOS 7. Archived versions aren't supported or tested.
+* A maintained version of CentOS 7. Archived versions aren't supported or tested
 * Red Hat Enterprise Linux 7.0, 7.1, 7.2, 7.3, or 7.4
```

Comment:
This should be 7.2+ IMO.  I don't think anyone is testing w/ 7.0 or 7.1, and the base kernels in those don't even work well w/ docker (I don't think DTR will even install).

Reply:
The compatibility matrix says that we support 7.1, but I'm not sure if it is correct. Either way the two docs need to be consistent.
https://success.docker.com/Policies/Compatibility_Matrix

Reply:
I've updated to point to compat matrix instead of duplicating